### PR TITLE
Add new users

### LINKED ISSUE #51: Interface to add users
Need interface to add users (current interface edits, deactivates, deletes)

### PR DESCRIPTION
**Pull request by dfaulken**
Closes #51.

Navbar button:
screen shot 2017-12-08 at 12 08 06 pm

Form:
screen shot 2017-12-08 at 12 08 09 pm

Form with an error:
screen shot 2017-12-08 at 12 09 51 pm

Successful save:
screen shot 2017-12-08 at 12 09 58 pm

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,10 +1,19 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  before_action :find_user, except: %i[import index]
+  before_action :find_user, except: %i[create import index new]
 
   before_action :restrict_to_staff
 
+  def create
+    @user = User.new
```
Comment (Anbranin) [inline app/controllers/users_controller.rb]: Does this need to be an instance variable?

Comment (Anbranin) [reply app/controllers/users_controller.rb]: Also, couldn't you just go `user = User.new user_params; if user.save....`

Comment (dfaulken) [reply app/controllers/users_controller.rb]: Because line 13 has a `render`.

Comment (Anbranin) [reply app/controllers/users_controller.rb]: OH

**Pull request merged**